Stripe/PublicHeaders/STPPaymentConfiguration.h

```diff
@@ -77,6 +77,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, assign, readwrite) STPShippingType shippingType;
 
+/**
+ The set of countries supported when entering an address. This property accepts
+ a set of ISO 2-character country codes.
+
+ The default value is nil, which will display all known countries. Setting this
+ property will limit the available countries to your selected set.
+ */
+@property (nonatomic, copy, nullable, readwrite) NSSet<NSString *> *availableCountries;
```

Comment:
what do you think about making this `null_resettable` and documenting that by default it's populated from `[NSLocale ISOCountryCodes]`?

Reply:
It looked more like this originally, but I was worried about inadvertently copying availableCountries as we pass it down the chain. It feels ugly to have to check something like `if (newAvailableCountries == setOfAllCountries) { _availableCountries = nil }` in every setter. But agreed that it probably makes more sense from an API perspective.